spec/fixtures/pineapple-lib/PineappleLib.podspec

```diff
@@ -0,0 +1,82 @@
+Pod::Spec.new do |s|
+  s.name         = 'PineappleLib'
```

Comment:
new spec fixture that sets up `app_host_name`. We did not have any fixtures that did that.